cli/args/mod.rs

```diff
@@ -1324,13 +1324,46 @@ impl StorageKeyResolver {
   }
 }
 
+fn expand_globs(paths: &[PathBuf]) -> Result<Vec<PathBuf>, AnyError> {
+  let mut new_paths = vec![];
+  for path in paths {
+    let path_str = path.to_string_lossy();
+    if path_str.contains('*') || path_str.contains('?') {
```

Comment:
One iteration instead of two:

```suggestion
    if path_str.chars().any(|c| matches!(c, '*' | '?')) {
```